### ISSUE TITLE
[Rogue] Cloak of Shadow & Evasion

### ISSUE THREAD

**Issue by Randomplayerx** [WoWManiaUK/Blackwing-Lair#3462]
**What should happen:**

Should share cooldown.

**Comment by Kitzunu** [comment]
Source of that? I have never seen that they should share CD? :o

**Comment by Kitzunu** [comment]
All i can find is 
 Patch 4.2.0 (2011-06-28): Cloak of Shadows cooldown is now 2 minutes, up from 90 seconds. In addition, Cloak of Shadows is no longer on the global cooldown.; Combat Readiness and Cloak of Shadows now share a cooldown.

**Comment by Randomplayerx** [comment]
@APTX4869s thanks for your linking this issue I don't see notification for new messages in the issues I made.

Btw I'll answer you cause Dev closed the issue.

1. Im re-editing my own message because I'm not confidence in my English which cause me to edit and remove/add things, there's nothing about "sure in what im saying" lol.

I already said I'm don't know more than him about how Ignite work, the Information in forums are sometimes false. the only way is to watch WoLs to produce it.

2. There indeed servers who not follow Twinstar I'm not making them "gods" and preach them.
Atlantiss.eu ingite was incorrect in the way it stacked Ignite they admitted it, other Cata servers has it incorrect too, just because it's "tricky" mechanic.
Let's not argue about it, I wrote that I just want to raise the awareness about Ignite, it's not so simple to fix specially in private server relay on sources that 'sometimes' are not so relayable.

3. Why are you trying to find logic in why I'm deleting my own messages lol?
I don't like making wars for nothing serious or ego shit.
Not everything I say is 100% true I don't have problems to admit it, this issue is just an example of it.

I dont understand czech aswell, but the Video that show 48k and the WoL in their Bug-report are two different things the WoL taken from Retail and Video from Twinstar PTR, It already explained why the Ignite is most correct in Twinstar. I wish Reign-wow would able to produce it better, I already explained the thing, whoever read it and understand it better. won't explain it again.
It's not about understanding the class >_<"

4. about not linking sources I'll be more cautious next time.

@Kitzunu Yeap what you linked is the correct I saw it too before I opened that topic but didn't paid attention, cause I felt comfort in myself. It should be with Combat Readiness indeed. Not with Envasion, this issue is closed. 
Addon showed me something different I re-checked it now it share cd with Cloak & Readiness which work as intent in the server.
